frontend/package.json

```diff
@@ -19,6 +20,7 @@
     "react-router-dom": "^5.3.0",
     "react-scripts": "4.0.3",
     "react-select": "^5.5.2",
+    "react-slippy-map": "^0.17.0",
```

Comment:
Uninstall this (if you switched to the other one) by running `npm uninstall react-slippy-map`